### ISSUE TITLE
Question: What is the effect of learning a bias and affine batch normalization on the results?

### ISSUE THREAD

**Issue by ofirzaf** [allenai/hidden-networks#6]
Hi,

In the paper you say that you don't learn an affine batch normalization (bias=0, scale=1) and from looking at the code you also don't learn a bias in convolution layers, but, you don't seem to have a good reason to do so.

I wonder what are the reasons you don't use a learned bias and batch norm in your experiments? 
does adding bias and affine batch norm improve or deteriorate the performance of the network?

Thanks

**Comment by mitchellnw** [comment]
Thanks for the question, it would probably improve the performance!

However, we are just trying to "train networks without changing any of the weights" and we didn't really have a good way of doing that for bias/batch norm. Definitely add if you're interested in improving accuracy.
